tools/data/build_audio_features.py

```diff
@@ -0,0 +1,265 @@
+import argparse
+import glob
+import os
+import os.path as osp
+import sys
+from multiprocessing import Pool
+
+import numpy as np
+from scipy.io import wavfile
+
+try:
+    import librosa
+    import lws
+    from tqdm import tqdm
+except ImportError:
+    print('Please import librosa, lws and tqdm first.')
+
+sys.path.append('..')
+
+
+class AudioTools:
+    """All methods related to audio feature extraction.
```

Comment:
It is better to give a reference/code link for the function details and official explanation